pyproject.toml

```diff
@@ -44,6 +44,7 @@ asynctest = { version = "^0.13.0",  python = "<3.8" }
 pytest = "^7.0.0"
 pytest-asyncio = "^0.19.0"
 pytest-cov = "^3.0.0 "
+dbus-fast = { version = "^1.83.0", markers = "platform_system == 'Linux'" }
```

Comment:
Can we avoid the duplicate requirement?

Reply:
Yes, its not actually needed since I changed the test. I forgot to revert the poetry change.

done in e2d7f2b48a1a3454a33e89c1194e6df8b98e046f